app/components/file/browse/file-explorer/file-explorer.html

```diff
@@ -1,6 +1,7 @@
 <bl-split-pane [config]="splitPaneConfig">
     <section blFirstPane class="tree-explorer-container" *ngIf="config.showTreeView">
         <bl-file-tree-view *ngFor="let source of this.workspace.sources;trackBy: trackSource"
+            [hideOnErrorCodes]="['NoPersistedOutput']"
```

Comment:
Not sure this belong here

Reply:
I would have this handler somewhere on the task code

Reply:
well i wanted something that was configurable, but i could just hard code it in the file-tree-view.component.ts

Reply:
i didn't want to over complicate it as it will probably be removed eventually.

Reply:
No I meant this should probably mot be at all in the file explorer code

Reply:
In the task file browser you could just not add the navigator to the workspace if container doesn't exist

Reply:
In the task file browser you could just not add the navigator to the workspace if container doesn't exist

Reply:
In the task file browser you could just not add the navigator to the workspace if container doesn't exist

Reply:
In the task file browser you could just not add the navigator to the workspace if container doesn't exist

Reply:
In the task file browser you could just not add the navigator to the workspace if container doesn't exist

Reply:
In the task file browser you could just not add the navigator to the workspace if container doesn't exist

Reply:
In the task file browser you could just not add the navigator to the workspace if container doesn't exist

Reply:
In the task file browser you could just not add the navigator to the workspace if container doesn't exist